addon/components/swiper-container.js

```diff
@@ -101,6 +101,11 @@ export default Component.extend({
       options.autoplayDisableOnInteraction = this.get('autoplayDisableOnInteraction');
     }
 
+    if (this.get('watchSlidesVisibility')) {
+      options.watchSlidesProgress = true;
```

Comment:
watchSlidesVisibility needs watchSlideProgress option turn on.


From the swiper api
![image](https://cloud.githubusercontent.com/assets/647691/25072276/1c4e527c-227f-11e7-9c88-2af5f2949032.png)